ppr-ui/src/components/registration/RegistrationBar.vue

```diff
@@ -4,7 +4,7 @@
       <v-col>
         <div class="actions">
           <v-btn
-            v-if="isMhr && (!isRoleQualifiedSupplier || isRoleStaff)"
+            v-if="isMhr && (!isRoleQualifiedSupplier || isRoleStaff || (isRoleManufactuer && hasMhrEnabled))"
```

Comment:
Although the right thought, i don't believe you need the `hasMhrEnabled` flag here. 
This was originally created when the only part of mhr included in PPR was for MHR Search. There is a little note in the default flag section in the utils file. 

`mhr-registration-enabled` would be the flag for Registrations, that being said, this flag currently is in place and will block the entire MHR Table Tab for anyone but Staff.

Reply:
Sounds good ill change it to not include hasMhrEnabled